lib/ClangImporter/ClangDerivedConformances.cpp

```diff
@@ -24,6 +24,42 @@
 using namespace swift;
 using namespace swift::importer;
 
+/// Known C++ stdlib types, for which we can assume conformance to the standard
+/// (e.g., std::map has template parameters for key and value types, and has
+/// members like key_type, size_type, and operator[]).
+enum class CxxStdType {
+  NotKnown,
```

Comment:
Nit: `NotKnown` is different capitalization than the rest of the values. Should we consider `unknown`? Or `uncategorized` (since we always know the actual type).

Reply:
My original intention was to use PascalCase to distinguish it from the snake_cased variants named after the stdlib types, but I realize there's not much a point to doing that.